src/main/java/nl/tudelft/contextproject/webinterface/WebServer.java

```diff
@@ -332,10 +293,8 @@ public static Cookie findCookie(String name, Cookie[] cookies) {
 	 * 		the newly created cookie
 	 */
 	public static Cookie createCookie(HttpServletRequest request) {
-		//Use the session to generate the Session ID.
```

Comment:
This comment sneakily explains why we have both a SESSION and a PERSISTENT cookie.

I will add a proper explanation why we use this cookie system in a different PR after this one has been merged.